synapse/cortex.py

```diff
@@ -396,7 +396,11 @@ def setFeedOffs(self, iden, offs):
         '''
         Return the list of splices at the given offset.
         '''
+        count = 0
         async for mesg in self.cell.layer.splices(offs, size):
+            count += 1
+            if not count % 1000:
```

Comment:
we should probably make this mod value smaller.  100?  thoughts?

Reply:
My reasoning is that this is streaming in key order. This should be the fastest thing lmdb can do.